### PR TITLE
TE-271 PropertySummary

### PR DESCRIPTION
**Pull request by Jonnygwi**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-271)

### What **one** thing does this PR do?
Creates component, examples and tests for `PropertySummary`

### Any other notes

- Extended Heading component with a `mini` size
- Removed unused Header variables from `header.variables`

image

### CODE REVIEW

**Comment by marcdavi-es** [comment]
@davidpelayo I paired on this so I shouldn't review. Please go wild... 😄

File: src/components/widgets/PropertySummary/component.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    </Segment.Group>
+  </Segment.Group>
+);
+
+Component.displayName = 'PropertySummary';
+
+Component.propTypes = {
+  /** The name of the location of the property. */
+  locationName: PropTypes.string.isRequired,
+  /** The price per night of the property, with currency symbol. */
```
Comment (davidpelayo) [inline src/components/widgets/PropertySummary/component.js]: The minimum price, should be 🏷

File: src/components/widgets/PropertySummary/component.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  nightPrice: PropTypes.string.isRequired,
```
Comment (davidpelayo) [inline src/components/widgets/PropertySummary/component.js]: The minimum night price, should be 🏷

File: src/components/widgets/PropertySummary/component.js (hunk 1 above)
Comment (davidpelayo) [reply src/components/widgets/PropertySummary/component.js]: screen shot 2018-04-10 at 3 28 19 pm
mmm, not sure?

**Pull request merged**